_rules/html-has-title-2779a5.md

```diff
@@ -38,6 +38,8 @@ Each target element has at least one [descendant](https://dom.spec.whatwg.org/#c
 
 **Note**: The `title` element exists in other namespaces such as SVG. These are not HTML `title` elements and should be ignored for this rule.
 
+**Note**: The [HTML 5.2 specificaion](https://www.w3.org/TR/html52/document-metadata.html#the-title-element) requires that a document only has one `title` element, and that it is a child of the `head` element of a document. However, HTMl 5.2 also describes what should happen in case of multiple titles, and titles outside the `head` element. Because of this, neither of these validation issues causes a conformance problem for WCAG.
```

Comment:
@kasperisager and you seem to disagree on whether we should cite HTML 5.2 or WHATWG LS. See my tentative summary of the situation at https://github.com/act-rules/act-rules.github.io/pull/768#discussion_r322190389

Thus said, since this question (5.2 vs WHATWG) is not solved and will anyway require to go through all rules to check and update them once a decision is taken, I see no reason to postpone merging this PR and revisit the rules later if we decide to cite WHATWG rather than 5.2